terragrunt/aws/scan_queue/cloudwatch_events.tf

```diff
@@ -0,0 +1,20 @@
+#
+# Event rule: defines events to capture and Step function to trigger
+#
+resource "aws_cloudwatch_event_rule" "sfn_events" {
+  name        = "state-machine-${var.env}-lock-cleanup"
+  description = "Capture State Machine events from the Scan queue step function"
```

Comment:
nit: could this be changed to something like `Capture failure State Maching events etc etc` to make it clearer?